lib/google/gax/grpc/stub.rb

```diff
@@ -29,16 +29,21 @@
 
 require "grpc"
 require "googleauth"
+require "google/gax/api_call"
 
 module Google
   module Gax
     module Grpc
       ##
-      # Stub
-      # TODO: Add documentation
-      module Stub
+      # Gax gRPC Stub
+      #
+      # This class wraps the actual gRPC Stub object and it's RPC methods.
+      #
+      class Stub
+        attr_reader :stub
```

Comment:
`wrapped_stub`? Might be a bit confusing for a "stub" to have a "stub" field.

Reply:
What about `grpc_stub`?